src/operator/src/insert.rs

```diff
@@ -576,6 +576,9 @@ impl Inserter {
 
                     for table in tables {
                         let table_info = table.table_info();
+                        if table_info.is_ttl_instant_table() {
```

Comment:
It's better to add a test.

Reply:
> It's better to add a test.

Might be a little tricky to test, is considering extending sqlness to test it but that might be the scope of another PR, syntax might be like this:
```sql
-- SQLNESS ARG use_http=true
-- SQLNESS ARG http_header="xxxx"
INSERT INTO xxx VALUES (...);
```

Reply:
Maybe you can try http test

https://github.com/GreptimeTeam/greptimedb/blob/main/tests-integration/tests/http.rs